reference-sheets/GitGuide.md

```diff
@@ -0,0 +1,81 @@
+## Introduction
+GitHub is a platform for managing and sharing code, built on top of Git, a version control system. It lets you track changes to your code, work on different features or fixes in separate branches, and merge everything together without losing progress. It’s especially useful for teams, making it easy to review code, resolve conflicts, and collaborate on projects efficiently. 
+
+This guide covers how to use GitHub to manage code effectively, best practices to keep in mind while using git, and a cheat sheet with all the important commands. It's also recommended that you checkout this [intro to git](https://developer.ibm.com/tutorials/d-learn-workings-git/) for a more indepth analysis on the inner workings of git. 
+
+## Best Practices
+A list of everything you should and are expected to do when working with git. 
+
+#### Use branches effectively
+Always create a new branch for each task, feature, or fix. This keeps your work isolated, makes collaboration easier, and helps prevent conflicts with the main branch.
+
+#### Merge regularly
+Merge changes back into the main branch often, once they’re tested and complete. Regular merging keeps the main branch current and helps avoid larger conflicts later on.
+
+#### Commit and push frequently
+Save and push your changes to the remote repository consistently. This keeps your progress backed up and ensures your team has access to the latest updates.
+
+#### Review code thoroughly
+Review and discuss changes before merging them into the main branch. This helps catch errors, improve code quality, and ensure everyone is aligned.
+
+## Cheat Sheet
+
+Simple list of all the most commonly used commands.
+
+#### Cloning
+`git clone <repository-link | repository-name> [repository-folder]`
+
+Copies the repo (repository) from github and puts it in a folder on your computer. It will put it in a folder with the name of the repository unless you specify a folder afterwards.
```

Comment:
Note that the default place of creation (no folder location arg) is the terminal's working directory.